### ISSUE TITLE
[Bug]: screenshot package needs update for flutter 3.22

### ISSUE THREAD

**Issue by sgehrman** [hm21/pro_image_editor#45]
### Package Version

22

### Flutter Version

3.22

### Platforms

Web

### How to reproduce?

install flutter 3.22, try to compile

```
Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
../../../../../.pub-cache/hosted/pub.dev/screenshot-2.3.0/lib/screenshot.dart:164:9:
Error: No named parameter with the name 'size'.
        size: logicalSize,
        ^^^^
```

### Logs (optional)

```sh
https://github.com/SachinGanesh/screenshot/issues/169
```

**Comment by hm21** [comment]
Thank you for reporting this issue. 

Currently, there is no newer version of the package `screenshot` available which resolve this issue. When it is released, I will update directly. 
For anyone running into this problem, you can fork this repository and replace the `screenshot` package inside `pubspec.yaml` like this:

```yaml
screenshot:
    git: 
       url: https://github.com/SachinGanesh/screenshot.git
```
In public packages, it is not possible to use packages directly from github, but for anyone who forks it and uses it themselves, this will work.

FYI: I'm making good progress with the new `Crop-Rotate-Editor` in the `dev branch`. This update will also remove the `screenshot` package, which will solve this problem directly.

**Comment by hm21** [comment]
Fixed in version >= 2.7.9.
